src/batch_triangular/Xhelper_funcs.ch

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
.ch extension file stands for cuda header, is that right?
.h is not enough?

Reply:
true, I tend to differentiate internal header files like this. KBLAS API has only .h header files.